src/vs/workbench/contrib/debug/browser/media/repl.css

```diff
@@ -35,7 +35,7 @@
 }
 
 .monaco-workbench .repl .repl-tree .output.expression.value-and-source .value {
-	flex: 1;
```

Comment:
Do we need a min-width for value? @isidorn

Reply:
I am fine either way, whatever you think behaves better.
I can also provide more feedback once I test this out

Reply:
I think we can keep it as is to avoid adding more constraints. Waiting for your feedback to see if it behaves as intended or if there are side effects that I haven't seen.